### ISSUE TITLE
Need examples of cross-cutting classifications (nodes not in dendrogram)  that are mapped up to CL in PCL.

### ISSUE THREAD

**Issue by dosumis** [obophenotype/CCN2#6]
I thought these would be easy to find. 

We used to map them here: 

https://github.com/obophenotype/brain_data_standards_ontologies/blob/master/src/dendrograms/taxonomy_details.yaml
Like this: 
```yaml
 non_taxonomy_roots: []
 # non_taxonomy_roots:
 #   - Node: CS201912131_128
 #     Cell_type: CL:4023011
 #     #Lamp5
 #   - Node: CS201912131_129
 #     Cell_type: CL:4023015
 #     #Sncg
 #   - Node: CS201912131_130
 #     Cell_type: CL:4023016
 #     #Vip
 #   - Node: CS201912131_131
 #     Cell_type: CL:4023017
 #     #Sst
 #   - Node: CS201912131_132
 #     Cell_type: CL:4023018
 #     #Pvalb
```

But then, IIRC, moved to doing in the class_curation pattern CSV.  Is that correct?  Checked some examples, but couldn't find.

**Comment by shawntanzk** [comment]
Yes, those were the ones before. So for GABAergic types - sst and pvalb were cross cutting and didn't have a node. However, after we started using markers on individual leaf nodes, the ontology classifies it to CL terms automatically, hence the curations were removed, and now the cross cutting terms (sst-like and pvalb-like) are not directly linked to CL term.

**Comment by shawntanzk** [comment]
Looked through mouse and don't see any cross-cutting classifications that are mapped to CL directly - there are stuff that are mapped through inference (see above). But yeah, because of 
https://github.com/obophenotype/brain_data_standards_ontologies/blob/master/src/templates/CCN202002013.tsv things get mapped well
I think you will find more in human and marmoset but that's a different dendrogram - not sure if you want to go there? happy to help if you are alright with examples from there

**Comment by dosumis** [comment]
Ahh - now I remember.  We really need to schedule some time to add better documentation to the BDSO repo.

Do we have any examples of cross-cutting classifications that are not marker-based?   e.g. are there any cross-cutting types defined based on Morph-Electric types?

**Comment by shawntanzk** [comment]
The best example are patchseq types that is cross cutting is L5 T-shaped Martinotti cell
![Screenshot 2022-09-20 at 09 22 33](https://user-images.githubusercontent.com/76212760/191206496-08d09dbb-e81b-49ad-bb38-ab75d880b44f.png)
They are split in the middle and are mapped to the CL term

**Comment by dosumis** [comment]
(BTW - do you know where we stashed the plots of marker expression that the mapping of expression to leaf nodes was based on?)

**Comment by shawntanzk** [comment]
huh strange - its in my local copy but seems to be excluded in github - will get them uploaded now

**Comment by shawntanzk** [comment]
https://github.com/obophenotype/brain_data_standards_ontologies/tree/master/src/patterns/data/expression_heatmaps
